src/components/equipment/CreateEditEquipment.tsx

```diff
@@ -196,7 +193,7 @@ export default function CreateEditEquipment() {
                   });
 
                   error
-                    ? enqueueError(error)
+                    ? enqueueError(error as ResourceId)
```

Comment:
The `error` is an arbitrary string containing the description of the error, not a ResourceId anymore

Reply:
Not sure how this works if it is implemented here on the scheduler

Reply:
I still see that on the scheduler-backend we have the old ResourceId as a reason returned

Reply:
Maybe there should be a separate task for this